### ISSUE TITLE
Addition of Multiple Images to (Merged) Cell

### ISSUE THREAD

**Issue by andrewbirchERMX** [ClosedXML/ClosedXML#1051]
- [] Bug
- [x] Feature

If you are logging a possible bug or feature request, please test with [the latest development build](https://ci.appveyor.com/project/ClosedXML/ClosedXML/branch/develop/artifacts) first.

**Version of ClosedXML**: 0.93.1

**What is the current behavior?**

Currently an image can only be attached to a specific cell, whether it forms part of a merged range or not. If multiple images are added, they stack on top of each other. They can only be offset at most by the width and/or height of one cell. There is no current way to offset an image within a merged cell. 

**What is the expected behavior or new feature?**

The ideal case would be to be able to add multiple images within a merged cell or row, as in the following image: 
![image](https://user-images.githubusercontent.com/37670177/47685589-979f9800-db9c-11e8-9521-b9e271b9935d.png)

**Did this work in previous versions of our tool?  Which versions?**

No

## Reproducibility
**Code to reproduce problem:**

Desired code to reproduce could be: 

```
        const string filePath1 = @"C:\Users\user\Pictures\Image1.PNG";
        const string filePath2 = @"C:\Users\user\Pictures\Image2.jpg";
        static void Main(string[] args)
        {
            IXLWorkbook workbook = new XLWorkbook(XLEventTracking.Disabled);
            var sheet = workbook.Worksheets.Add("TestSheet");
            sheet.Range("A1:G1").Merge();
            sheet.AddPicture(filePath1).WithPlacement(XLPicturePlacement.FreeFloating).WithSize(150, 150).MoveTo(sheet.Cell("A1").Address);
            using (var stream = new FileStream(filePath1, FileMode.Open))
            {
                sheet.AddPicture(stream, XLPictureFormat.Jpeg).WithPlacement(XLPicturePlacement.FreeFloating).WithSize(150, 150).MoveTo(sheet.Cells("A1:G1").Address, 0,0);
            };
            using (var stream = new FileStream(filePath2, FileMode.Open))
            {
                sheet.AddPicture(stream, XLPictureFormat.Jpeg).WithPlacement(XLPicturePlacement.FreeFloating).WithSize(150, 150).MoveTo(sheet.Cells("A1:G1").Address, 150, 0);
            };
            workbook.SaveAs(@"Insert File Path");
        }
```

- [x ] I attached a sample spreadsheet.  (You can drag files on to this issue)
[PruebaPics.xlsx](https://github.com/ClosedXML/ClosedXML/files/2527392/PruebaPics.xlsx)

**Comment by igitur** [comment]
Can you please create a file in Excel of what you want to achieve and upload it here?

**Comment by andrewbirchERMX** [comment]
Hi @igitur, PFA the attached Demo file: 

[FotosMergedCellsDemo.xlsx](https://github.com/ClosedXML/ClosedXML/files/2530780/FotosMergedCellsDemo.xlsx)

**Comment by igitur** [comment]
Ah I see. This seems to be a limitation of the file format. For now, you'll have to manually calculate the closest cell to the left of the image and the offset from that cell's edge.

**Comment by andrewbirchERMX** [comment]
Ah ok, I'll try that for now I guess. Thanks for your help!

**Comment by andrewbirchERMX** [comment]
Hi I've been trying to do the calculation manually, but have an issue regarding the conversion of pixels (for the image size) to cell width (inch/cm) to get the closest cell. I googled around awhile and found the following issue: [#510](https://github.com/ClosedXML/ClosedXML/issues/510)

I see you mentioned moving dimensions from pixels to in/cm. Did that idea go any further?

**Comment by igitur** [comment]
No, haven't really looked at that further. There didn't seem to be much of a need, but maybe there is now.

**Comment by igitur** [comment]
@andrewbirchERMX Check out my comment at https://github.com/ClosedXML/ClosedXML/issues/510#issuecomment-435089954

**Comment by andrewbirchERMX** [comment]
Great, I'll check it out, thanks a lot!
